cmab/agent.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List
+
+from bandit import Bandit
+
+
+class Agent:
+    __bandit_names = ('spg', 'light_tank', 'heavy_tank', 'medium_tank', 'at_spg')
+
+    def __init__(self, name: str, num_turns: int):
+        self.__agent_name: str = name
+        self.__bandits = {name: Bandit(name, num_turns) for name in Agent.__bandit_names}
```

Comment:
Ok, will do

Reply:
If you don't mind me asking, why? wouldn't it make sense to declare the names just once rather than every time the class is instantiated?

Reply:
I'm telling not about making __bandit_names instance attribute. It may stay class attribute but I think it is better to access it via self rather than via class.

Reply:
Aaaa, I see